scalafix-tests/input/src/main/scala/test/DisableSyntax.scala

```diff
@@ -9,16 +9,40 @@ DisableSyntax.keywords = [
 DisableSyntax.noTabs = true
 DisableSyntax.noSemicolons = true
 DisableSyntax.noXml = true
+DisableSyntax.regex = [
+  {
+    pattern = "[P|p]imp"
+    message = "Please consider a less offensive word such as Extension"
+  }
+  "Await\\.result"
+]
 */
 package test
 
+import scala.concurrent._
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext.Implicits.global
+
 case object DisableSyntax {
+
+
+  null /* assert: DisableSyntax.keywords.null
+  ^
+  null is disabled
+  */
+
   var a = 1                 // assert: DisableSyntax.keywords.var
-  null                      // assert: DisableSyntax.keywords.null
   def foo: Unit = return    // assert: DisableSyntax.keywords.return
   throw new Exception("ok") // assert: DisableSyntax.keywords.throw
 
   "semicolon";              // assert: DisableSyntax.noSemicolons
   <a>xml</a>                // assert: DisableSyntax.noXml
 	                          // assert: DisableSyntax.noTabs
+
+  implicit class StringPimp(value: String) { // assert: DisableSyntax.[P|p]imp
```

Comment:
Hmm, I wonder if `CustomMessage` should have an optional `id` field to avoid exposing complex messages like this into linter messages. Opened https://github.com/scalacenter/scalafix/issues/495